contracts/v0.2/src/Feed.sol

```diff
@@ -8,41 +8,30 @@ import {IFeed} from "./interfaces/IFeed.sol";
  * @title Orakl Network Feed
  * @author Bisonai Labs
  * @notice A contract that stores the historical and latest answers, and
- * the timestamp submitted by a whitelisted set of oracles. The
- * contract owner can add or remove oracles from the whitelist using
- * `changeOracles` function.
- * @dev A set of oracles is represented with `oracles` list and
- * `whitelist` mapping. The submitted answers are expected to be
- * submitted directly by whitelisted EOA oracles or through a
+ * the timestamp submitted by oracle.
+ * @dev The submitted answers are expected to be submitted through a
  * `SubmissionProxy` contract.
  */
 contract Feed is Ownable, IFeed {
-    uint8 public override decimals;
-    string public override description;
+    uint8 public decimals;
+    string public description;
+    address public oracle;
```

Comment:
If in our usecase, if oracle is always submission proxy contract address, how about using name `submissionProxy` instead of `oracle`?

Reply:
It is just our internal assumption that it will be submitted by `SubmissionProxy`, however, it does not always have to be `SubmissionProxy`. The behavior of the contract is not dependent on it.

Reply:
If it works as our internal assumption, I think it is much clear to set address variable name as `submissionProxy` and modifier as `onlySubmissionProxy`. Are there any other cases which you expect?

Reply:
I think the word `oracle` has a more general meaning than submission proxy. In the way how we use it, it is only address without any functionality, and represents whitelisted EAO or contract that can submit. It does not have to be always used with `SubmissionProxy`, if we move proof of reserve to v0.2 it would probably submit directly without proxy unless backend is updated.